lib/workers/global/index.ts

```diff
@@ -63,10 +63,12 @@ function checkEnv(): void {
   const range = pkg.engines!.node!;
   const rangeNext = pkg['engines-next']?.node;
   if (process.release?.name !== 'node' || !process.versions?.node) {
-    logger.warn(
-      { release: process.release, versions: process.versions },
-      'Unknown node environment detected.'
-    );
+    if (!process.env.RENOVATE_X_IGNORE_NODE_WARN) {
+      logger.warn(
```

Comment:
we should log a info message then, so we see it if users posting log messages .

probably always log a info message about used node version for easier issue tracking?

Reply:
Done